sirepo/package_data/static/js/radia.js

```diff
@@ -1221,13 +1224,15 @@ SIREPO.app.directive('bevelTable', function(appState, panelState, radiaService)
                 <col style="width: 20ex">
                 <col style="width: 20ex">
                 <col style="width: 20ex">
+                <col style="width: 20ex">
```

Comment:
Could do this in fewer lines with `<col span="4" style="width: 20ex">`. Also, just wondering why there are 4 of these but 5 columns (not including edit button). I guess the last one would be redundant since it can just size to whatever is left?

Reply:
[752caf2](https://github.com/radiasoft/sirepo/pull/4699/commits/752caf200761300ff36e71c8dd89b4553dd3782e)
(4 instead of 5 was a mistake)